### PR TITLE
Feature/top level notifications

### LINKED ISSUE #124: [UI Core library] Top Level Notification
https://www.figma.com/file/8mVm6LTbp2Z0RaWWjTZoft/%F0%9F%90%9DHIVE-Hazelcast-Design-System?node-id=1735%3A1325

Related to Alerts and Toast Notifications.

### PR DESCRIPTION
**Pull request by jankoritak**
Closes #124

### CODE REVIEW

File: packages/ui/src/Notification.module.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  .content {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: center;
+    /*
+     * Note:
+     * The (min)height is equal to corresponding icon height. Necessary due to 'flex-start' alignment and
+     * specific alignment differences between single-line and multi-line text content.
+     */
+    min-height: c.$grid * 6;
```
Comment (aigoncharov) [inline packages/ui/src/Notification.module.scss:25]: What happens if we remove the `min-height`?

**Review by aigoncharov — APPROVED** [review]
LGTM with one outstanding question!

File: packages/ui/__stories__/Notification.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/8mVm6LTbp2Z0RaWWjTZoft/%F0%9F%90%9DHIVE-Hazelcast-Design-System?node-id=1735%3A1455',
+  },
+}
+
+export const SuccessWithLongText = () => <Notification type="success">{longText}</Notification>
+
+export const SuccessWithLink = () => (
+  <Notification type="success" link={link} linkHref={linkHref}>
```
Comment (danad02) [inline packages/ui/__stories__/Notification.stories.tsx:28]: The chevron is a bit shifted for all links:
image

**Review by danad02 — APPROVED** [review]

File: packages/ui/src/Notification.module.scss (hunk 1 above)
Comment (jankoritak) [reply packages/ui/src/Notification.module.scss:25]: Content is not vertically aligned properly.

Screenshot 2021-01-04 at 11 04 39

**Comment by jankoritak** [comment]
verify

**Comment by aigoncharov** [comment]
@jankoritak could you take a look at it on Monday?

**Pull request merged**